Gemfile

```diff
@@ -25,6 +25,14 @@ group(:development, :test) do
   gem 'cheffish', "~> 1.3", "!= 1.3.1"
 
   gem 'ruby-shadow', :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+
+  # For external tests
+  gem 'chefspec', github: 'jkeiser/chefspec', branch: 'jk/chefspec-12.5'
+  gem 'chef-sugar'
+  gem 'poise', github: 'poise/poise', branch: 'deeecb890a6a0bc2037dfb09ce0fd0a8931519aa'
```

Comment:
Halite tests against Chef master nightly (eg. https://travis-ci.org/poise/halite/jobs/82850534) and on every commit, so I think it should be working. Can you send me the errors? It is sensitive to odd bundler setups because of the way it hooks in to Rubygems so that might be related.